src/Response/Body.php

```diff
@@ -4,38 +4,29 @@
 
 use Psr\Http\Message\StreamInterface;
 use Jasny\HttpMessage\Stream;
+use Jasny\HttpMessage\Message;
```

Comment:
We register the namespace rather than the trait itself to prevent naming conflict. (This is preferable above using `as` for a alias)